model/project.rb

```diff
@@ -128,6 +128,10 @@ def validate
     end
   end
 
+  def default_private_subnet(location)
+    private_subnets_dataset.where(location: location).where(Sequel[:private_subnet][:name] => "default-#{LocationNameConverter.to_display_name(location)}").first
+  end
```

Comment:
nit: since we expect `default_private_subnet` to return the default private subnet, we can also assemble of it to there. It allows to use the same name variable too.
```suggestion
  def default_private_subnet(location)
    name = "default-#{LocationNameConverter.to_display_name(location)}"
    ps = private_subnets_dataset.where(location: location).where(Sequel[:private_subnet][:name] => name).first
    ps || Prog::Vnet::SubnetNexus.assemble(project_id, name: name, location: location).subject
  end
```